packages/editor/src/components/block-list/block.js

```diff
@@ -357,6 +365,12 @@ export class BlockListBlock extends Component {
 		}
 	}
 
+	forceFocusedContextualToolbar() {
+		this.isForcingContextualToolbar = true;
+		// trigger a re-render
+		this.setState( ( prevState ) => prevState );
```

Comment:
State is applied as a patch. We don't need to return anything, and by doing so we might actually be causing more work to be performed.

Should be sufficient:

```js
this.setState( () => ( {} ) );
```

See also: https://github.com/WordPress/gutenberg/blob/4369d93f283f5203ad4a91ad04cce015b2699654/packages/data/src/components/with-select/index.js#L127

Reply:
Improved 👍